modules/kubernetes/grafana-agent/templates/operator-values.yaml.tpl

```diff
@@ -0,0 +1,9 @@
+resources:
+  requests:
+    cpu: 25m
+    memory: 100Mi
+tolerations:
+  - operator: "Exists"
```

Comment:
We don't need tolerations for the operator it self. We will always have standard nodes without tolerations that we can use.